### DISCUSSION TITLE
Resources and Issues before applying to GSOC '23

### DISCUSSION THREAD

**Discussion by AnuragMaiti** [openvinotoolkit/openvino#15956] [Google Summer of Code]
Dear OpenVINO Developers,

I am Anurag Maiti, a first-year CSE undergrad student from BITS Pilani, Hyderabad Campus. Before applying for GSOC '23 here, I would like to participate in a few small issues to gain some experience in the field and further enhance my knowledge of machine learning and AI. I plan to finish the pre-requisite task in the next week or two and would love to get in touch with the mentors!

Are there any resources I can access or issues that I can work on while I am completing the pre-requisite task? Also, I am interested in the project: ```Add ensembling methods for tiling to Anomalib``` whose mentors are ```Dick Ameln and Samet Akcay```. Is there any way I can get in touch with them?

Looking forward to working with all of you!

Yours Sincerely,
Anurag.

**Comment by samet-akcay** [comment]
Hi @AnuragMaiti, thanks for your interest in the project(s)! I am one of mentors of this project. You could use the discussions sections of the [OpenVINO Notebooks](https://github.com/openvinotoolkit/openvino_notebooks/discussions/categories/gsoc-2023) or [Anomalib](https://github.com/openvinotoolkit/anomalib/discussions/categories/google-summer-of-code-gsoc-2023) repositories, or alternatively contact [me](samet.akcay@intel.com) or [@djdameln](dick.ameln@intel.com) via email. Hope this would be useful for you!

**Reply by AnuragMaiti** [reply]
Thank you so much, @samet-akcay! I'll be sure to check out the discussions and raise any queries as and when it is needed.
